package.json

```diff
@@ -1,10 +1,59 @@
 {
   "name": "logdna-s3",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
again, we should get this automated and on Jenkins (example from [logdna-bunyan](https://github.com/logdna/logdna-bunyan/commit/a141f9fb504b3ab656d58266b400e4ed26fec5e4)). doesn't need to happen here.


might be worth considering something like [`claudia`](https://github.com/claudiajs/claudia) for the build step. [`claudia pack`](https://github.com/claudiajs/claudia/blob/master/docs/pack.md) provides a standard way to generate a zip file like we're already doing, but does a lot more.

Reply:
@mdeltito, we have a separate ticket (#4) for this; do you want me to implement that within this PR or in separate PR after finishing this?

Reply:
no, as long as we're tracking a ticket to do it after the refactor, that seems good to me